linchpin/error_handlers.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
```

Comment:
The linchpin/exceptions namespace is where exceptions/errors go.

Reply:
Sounds good, I'll move it